integrations/larky/input/query_string/test-query-string.star

```diff
@@ -0,0 +1,50 @@
+load("@stdlib//unittest", "unittest")
+load("@vendor//asserts", "asserts")
+load("@stdlib//hashlib", "hashlib")
+load("@stdlib//binascii", "binascii")
+load("@vgs//vault", "vault")
+load("@vgs//http/request", "VGSHttpRequest")
+load("@stdlib//base64", base64="base64")
+load("@vendor//Crypto/Hash/HMAC", HMAC="HMAC")
+load("@vendor//Crypto/Hash/SHA256", SHA256="SHA256")
+load("@stdlib//builtins", builtins="builtins")
+load("@stdlib//json", json="json")
+load("@stdlib//codecs", codecs="codecs")
+
+vault = {
+    "tok_sandbox_a4JULYBMphkeL2iV7ipwdL": "4111111111111111"
+}
+
+def process(input, ctx):
+    path = input.path
+    query = input.query_string
+    body = json.loads(str(input.body))
+
+    lst = query.split('=')
```

Comment:
Do we have a way of getting a key/value representation of the query string? This example is assuming that the query string keys are always in a specific order.

Reply:
came up with https://stackoverflow.com/questions/21584545/url-query-parameters-to-dict-python